### ISSUE TITLE
is it possible to get embedding of last layer of network and add softmax to get a Classifier

### ISSUE THREAD

**Issue by roodkcab** [FoundationVision/VAR#21]

**Comment by keyu-tian** [comment]
hi @roodkcab, maybe you can create a classifier `self.head2 = nn.Linear(self.C, classification_classes)` and replace `self.head(self.head_nm(h.float(), cond_BD).float()).float()` with `self.head2(self.head_nm(h.float(), cond_BD).float().mean(dim=1)).float()` in `models/var.py` to get classification logits.

**Comment by roodkcab** [comment]
great! I'll have a look. really impressive work! thanks again.
